lib/src/DynamicProto.ts

```diff
@@ -161,9 +173,8 @@ function _getObjProto(target:any) {
  */
 function _forEachProp(target: any, func: (name: string) => void) {
     let props: string[] = [];
-    let getOwnProps = Obj["getOwnPropertyNames"];
```

Comment:
Just moved this so it's only done once instead of every call as it won't change.